Documentation/manual-install.md

```diff
@@ -38,6 +38,8 @@ $ export METERING_NAMESPACE=metering-$USER
 $ ./hack/openshift-install.sh
 ```
 
+[Note: the default has no storage setup so you may want to choose HDFS, do so with: ```export METERING_CR_FILE=manifests/metering-config/hdfs-storage.yaml```, you *must* choose some storage option for a successful install.]
```

Comment:
This renders weird - might want to just remove the starting and ending brackets, and replace with `**Note**`

Reply:
the default `MeteringConfig` CR has no storage ...

Reply:
👍 changed in fe2da36 and 7a63280